models/app.model.js

```diff
@@ -64,4 +54,13 @@ exports.updateVotes = (article_id, inc_votes) => {
   return db.query(`UPDATE articles SET votes = votes + $2 WHERE article_id = $1 RETURNING *`, [article_id, inc_votes]).then(({ rows }) => {
     return rows[0];
   })
+}
+
+exports.selectCommentAndDelete = (comment_id) => {
```

Comment:
This name doesnt seem appropriate as you are not selecting a comment here but Just deleting it